### PR TITLE
Migrate readme yaml

### PR DESCRIPTION
**Pull request by vadim-gleyf**
## what
* Add `README.yaml`

## why
* Standardize README

### CODE REVIEW

File: .build-harness [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export SHELL = /bin/bash
```
Comment (sarkis) [inline .build-harness]: This file and the build-harness/ directory should be in `.gitignore` and not committed into the repo.

**Review by sarkis — DISMISSED** [review]
Need to add `build-harness` and `.build-harness` to `.gitignore` i.e. https://github.com/cloudposse/terraform-aws-ecs-container-definition/blob/master/.gitignore#L10 and remove these from version control.

**Review by osterman — CHANGES_REQUESTED** [review]
Agree with @sarkis

File: README.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    cluster_name     = "us-east-1"
+    parent_zone_name = "cloudxl.net"
+    zone_name        = "$${name}.$${parent_zone_name}"
+    region           = "us-east-1"
+  }
+  ```
+
+  <br/>
+
+  ![kops-state-backend](images/kops-state-backend.png)
```
Comment (osterman) [inline README.yaml]: Move this to `screenshots` with a description of "Example of outputs from the module after running `terraform apply`"

**Review by osterman — CHANGES_REQUESTED** [review]
See inline comments

File: README.md [hunk 3]
```diff
@@ -71,50 +93,95 @@ aws route53 list-tags-for-resources --resource-type hostedzone --resource-ids Z2
...
 
 
 ## Help
 
 **Got a question?**
 
-File a GitHub [issue](https://github.com/cloudposse/terraform-aws-kops-state-backend/issues), send us an [email](mailto:hello@cloudposse.com) or reach out to us on [Gitter](https://gitter.im/cloudposse/).
+File a GitHub [issue](https://github.com/cloudposse/terraform-aws-kops-state-backend/issues), send us an [email][email] or join our [Slack Community][slack].
+
+## Commerical Support
```
Comment (osterman) [inline README.md]: Rebuild

**Review by osterman — APPROVED** [review]
LGTM, but rebuild one more time to fix README typo.

**Review by osterman — CHANGES_REQUESTED** [review]
Rebuild

**Review by osterman — APPROVED** [review]

**Pull request merged**